### PR TITLE
Added blog-post on nuisance parameter elimination.

### PR DESCRIPTION
**Pull request by gregoryross**
I've added my blog-post. It's the first time I've used this system so there may be something wrong with my formatting.

### CODE REVIEW

File: _posts/2016-09-29-eliminating-nuisance-parameters.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+layout: post
+author: 
+  - gregoryross
+date: 2016-09-28
+title: Integrated likelihood methods for eliminating nuisance paramaters
+published: true
+---
+
+The process of inferring conclusions from experimental observations often requires the construction and evaluation of a likelihood function, which, gives the probability, or probability density, of the observed data given a particular statistical model.
```
Comment (maxentile) [inline _posts/2016-09-29-eliminating-nuisance-parameters.md]: ", which," --> ", which"

File: _posts/2016-09-29-eliminating-nuisance-parameters.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+The likelihood function forms the basis for many statistical inference techniques. For example, one could estimate the parameters of a model by maximizing the likelihood. The problem is that likelihood functions frequently contain more parameters than we care about. The inexplicable oruninteresting
```
Comment (maxentile) [inline _posts/2016-09-29-eliminating-nuisance-parameters.md]: "oruninteresting" --> "or uninteresting"

File: _posts/2016-09-29-eliminating-nuisance-parameters.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+samples drawn from a binomial distribution. This example is explored in [this Jupyter notebook](https://github.com/gregoryross/Walkthroughs/blob/master/Integrated_Likelihoods/Nuisance_paramaters_in_likelihoods.ipynb).
+
+
+## Estimation with binomial samples
+
+Imagine that we’ve been challenged by a friend to estimate the number of times a coin has been flipped in a series of independent sessions, but(!) we’re only told how many times the coin landed on heads. In each session, the coin is flipped N times, heads appears with a probability p, and both are
+unknown. How can we estimate N? The binomial distribution is the natural probabilistic model for this situation, and we can use it to construct a likelihood function for N given the number of times heads is drawn with p as the nuisance parameter. The example in the paper considers five sessions,
+with the number of heads being 16, 18, 22, 25, and 27 for each session. This example, including equations and code, is worked through in detail here [hyperlink]. For this case, the full two-dimensional likelihood surface looks like this:
```
Comment (maxentile) [inline _posts/2016-09-29-eliminating-nuisance-parameters.md]: "[hyperlink]"?

**Review by maxentile — COMMENTED** [review]
This looks great to me, Greg! Spotted a couple typos, but I think this is very clearly described and illustrated.

**Comment by gregoryross** [comment]
@maxentile Thanks for reading it over!

**Comment by jchodera** [comment]
Awesome!

Will merge/post/tweet it in the morning!

**Comment by jchodera** [comment]
@gregoryross : The images aren't rendering for some reason. Can you actually put those in this repo and link them here?

**Pull request merged**